packages/react/src/components/Forms/DatePicker/CalendarCell.tsx

```diff
@@ -0,0 +1,36 @@
+import React, { useRef } from "react";
+import classNames from "classnames";
+import { useCalendarCell } from "@react-aria/calendar";
+import { CalendarDate } from "@internationalized/date";
+import { CalendarState } from "@react-stately/calendar";
+import { Button } from ":/components/Button";
+
+interface CalendarCellProps {
+  state: CalendarState;
+  date: CalendarDate;
+}
+
+export const CalendarCell = ({ state, date }: CalendarCellProps) => {
+  const ref = useRef<HTMLButtonElement>();
+  const { cellProps, buttonProps, isSelected, formattedDate } = useCalendarCell(
+    { date },
+    state,
+    ref
+  );
+  return (
+    <td {...cellProps}>
+      <Button
+        size="small"
```

Comment:
I can make the grid narrower, np ! And increase margin inside clickable elements

Reply:
There are a lot of empty spaces because of the menu's header width. Actually, in some calendar, month name and year are wider than the figma design. To avoid having next/prev buttons moving on month changes, I have fixed the width of the two dropdown elements.